src/services/auth/auth.service.ts

```diff
@@ -23,6 +24,7 @@ export async function authenticate(
   })
```

Comment:
No reason, because at that time i was coding, I was sick of Services as classes hahaha. But I will update to a class, just to keep up with the project syntaxe.